docs/admin/high-availability/building.md

```diff
@@ -209,14 +209,16 @@ continuously try to add itself to the list of endpoints while removing
 the other ones, causing a lot of extraneous updates in kube-proxy
 and other components.
 
-Starting with Kubernetes 1.9, a new reconciler implementation is available.
+Starting with Kubernetes 1.9, a new alpha reconciler implementation is available.
 It uses a *lease* that is regularly renewed by each apiserver
 replica. When a replica is down, it stops renewing its lease, and
 the other replicas notice that the lease expired and remove it
 from the list of endpoints. You can switch to the new reconciler
 by adding the flag `--endpoint-reconciler-type=lease` when starting
 your apiserver replicas.
 
+Note: The lease reconciler is in alpha.

```

Comment:
appreciate referencing this as alpha, but could we use https://kubernetes.io/docs/home/contribute/includes/#feature-state to indicate it?

Reply:
can the alpha include be included within a sub-section of a document? From what I can tell it applies to the entire section.